github-actions/utils.ts

```diff
@@ -0,0 +1,18 @@
+import {getInput} from '@actions/core';
+const {getToken} = require('github-app-installation-token');
+
+export async function getAuthTokenForAngularRobotApp(): Promise<string>;
+export async function getAuthTokenForAngularRobotApp(
```

Comment:
Nit: Could you add a description? I specifically would be interested in how this token is preferred, compared to the token that is already available in Github actions? i.e. do we want to act as the ng robot, compared to the `github-actions` bot account?